### DISCUSSION TITLE
Issue Installing

### DISCUSSION THREAD

**Discussion by Gharla2Sami** [laravel-frontend-presets/tall#105] [General]
When installing tall stack preset library I face a lot of issues. I have try to resolve this many times but I'm not able to figure out what is really the problem

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires laravel-frontend-presets/tall ^4.0 -> satisfiable by laravel-frontend-presets/tall[v4.0.0].
    - laravel-frontend-presets/tall v4.0.0 requires illuminate/support ^8.0 -> found illuminate/support[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require livewire/livewire:*" to figure out if any version is installable, or "composer require livewire/livewire:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by josefernandoperezgarcia1998** [comment]
Did u install composer? And do "composer install"?

**Reply by Gharla2Sami** [reply]
Yes I did.

**Reply by Gharla2Sami** [reply]
Problem 1
    - Root composer.json requires laravel-frontend-presets/tall ^4.0 -> satisfiable by laravel-frontend-presets/tall[v4.0.0].
    - laravel-frontend-presets/tall v4.0.0 requires illuminate/support ^8.0 -> found illuminate/support[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

**Reply by Gharla2Sami** [reply]
This package is yet compatible with laravel 9. I just install it with laravel 8 and it work perfectly.

**Reply by ifathurrohman** [reply]
> 

Absolutely yess

**Reply by danharrin** [reply]
It's compatible now
